### PR TITLE
Add Spanish translations of docs

### PR DESCRIPTION
**Pull request by virejdasani**
### Description

Thanks for contributing this Pull Request. Make sure that you submit this Pull Request against the `master` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

Translated the docs to Spanish.

- Relevant Issues : #157 
- Relevant PRs : (optional)
- Type of change :
  - [ ] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [x] Addition or Improvement of documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,9 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
   <a href="https://github.com/intuit/Trapheus/releases"><img src="https://img.shields.io/github/v/release/intuit/trapheus.svg" alt="release badge"/></a>
 </p>
 
- <img src="https://ch-resources.oss-cn-shanghai.aliyuncs.com/images/lang-icons/icon128px.png" width="22px" /> [English](README.md) |  [简体中文](./docs/README.zh-CN.md) | [français](./docs/README.fr.md)
-
-
-* **Important:** this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS  pricing page](https://aws.amazon.com/pricing/) for details.
+<img src="https://ch-resources.oss-cn-shanghai.aliyuncs.com/images/lang-icons/icon128px.png" width="22px" /> [English](README.md) | [简体中文](./docs/README.zh-CN.md) | [français](./docs/README.fr.md) | [español](./docs/README.es.md)
 
+- **Important:** this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS pricing page](https://aws.amazon.com/pricing/) for details.
```
Comment (namitad) [inline README.md:19]: can we remove all the unrelated changes from this file. @virejdasani your required changes seem to be in this line https://github.com/intuit/Trapheus/pull/158/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R17 , would my understanding be correct?

File: docs/README.es.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+         - VPC: Trapheus-VPC-[región]
+         - Zona de disponibilidad: elige una
+         - Bloque CIDR IPv4: 10.0.0.0/19
+       - Subred2:
+         - VPC: Trapheus-VPC-[región]
+         - Zona de disponibilidad: elija una diferente a la Subred1 AZ.
+         - Bloque CIDR IPv4: 10.0.32.0/19
+     - Ha creado una VPC con solo dos subredes privadas. Si está creando subredes no privadas, verifique [la relación entre subredes públicas y privadas, subred privada con ACL de red personalizada dedicada y capacidad adicional](https://aws-quickstart.github.io/quickstart-aws-vpc/) .
+
+5. Una o más instancias de una base de datos RDS que desea restaurar.
```
Comment (namitad) [inline docs/README.es.md:70]: the translated file seems incomplete. could you check?

**Review by namitad — CHANGES_REQUESTED** [review]
@virejdasani please address the comments

**Comment by stationeros** [comment]
@virejdasani I am going to cut a release this week , do let us know if you can address the comments by then

**Comment by stationeros** [comment]
Closing this owing no response for a month

**Pull request closed without merging**